core/templates/dev/head/pages/exploration_editor/EditorNavigationDirective.js

```diff
@@ -63,6 +63,8 @@ oppia.directive('editorNavigation', [
           $scope.showUserHelpModal = function() {
             var explorationId = ContextService.getExplorationId();
             SiteAnalyticsService.registerClickHelpButtonEvent(explorationId);
+            var editorTutorialMode = 'editor';
```

Comment:
EDITOR_TUTORIAL_MODE, TRANSLATION_TUTORIAL_MODE

Reply:
ok will change it to that.

Reply:
Done